### PR TITLE
turing -> hc rebranding: boba-community (esp. Starter project)

### PR DESCRIPTION
**Pull request by wsdt**
:clipboard: Addresses #457 

## Overview

Refactor `boba_community` projects, especially TuringStarter & redeploy on Goerli.
Deploy faucet on Goerli. 
Adapted metaTx api for Goerli support. 

## Changes

- Rename folders + files in boba_community
- Refactor code (TuringStarter)
- Deploys of HcStarter + Faucet on Goerli
- Deployment & Adaption of new metatx api for Goerli

## Testing

- Rebuilt the full stack (success)
- Rebuild all packages in turing-start
- Run of TuringStarter React app & smart contract tests work
- Faucet tests run

### CODE REVIEW

**Review by InoMurko — APPROVED** [review]

**Comment by wsdt** [comment]
will also add the Goerli config in this PR, please don't merge yet. 

@Boyuan-Chen @InoMurko Goerli Blockexplorer, Gateway & Faucet don't seem to be live yet. What is the current status? Could someone send me some ETH if that's true, so that I can deploy the contracts?

**Review by souradeep-das — APPROVED** [review]

**Comment by wsdt** [comment]
@InoMurko @souradeep-das would be ready to merge. 
Would be great if you could have a look at it again, many things changed.

File: boba_community/hc-start/packages/dapp-contracts/.openzeppelin/unknown-2888.json [hunk 1]
```diff
@@ -0,0 +1,128 @@
+{
```
Comment (souradeep-das) [inline boba_community/hc-start/packages/dapp-contracts/.openzeppelin/unknown-2888.json]: should we add these to gitignore?

File: boba_community/hc-twitter/README.md [hunk 2]
```diff
@@ -1,10 +1,15 @@
-# Turing KYC example
+# Turing Faucet
```
Comment (souradeep-das) [inline boba_community/hc-twitter/README.md]: hc 😄

**Review by souradeep-das — APPROVED** [review]
new changes looks good to me 🚀

**Pull request merged**